tests/acceptance/crate-test.js

```diff
@@ -66,7 +66,7 @@ module('Acceptance | crate page', function(hooks) {
     await visit('/');
     await click('[data-test-just-updated] [data-test-crate-link="0"]');
 
-    assert.equal(currentURL(), '/crates/nanomsg');
+    assert.equal(currentURL(), '/crates/nanomsg/6.8.9');
```

Comment:
Is this test one that's failing for you locally? I think it's because this test uses the factories, and the version number isn't guaranteed to be the same every time the factory gets called?

This can be fixed by explicitly setting a `newest_version` for this particular instance, sort of like [this test explicitly sets a `max_version`](https://github.com/rust-lang/crates.io/blob/3e5dd28127284e64ff8e4b106ca5dc3472398e5a/tests/acceptance/crate-test.js#L29).

Reply:
Running the tests locally crashes fairly badly. I don't have much experience with npm stuffs, so I don't have much clue what is going on. It _did_ work for me. I don't know of having changed anything (although I'm a compulsive `brew upgrade`r and that might've broken something).

Reply:
Yikes! I have occasional luck with removing `node_modules` and re-`npm install`ing? We can look at it tonight though :)

Reply:
Locally, I've replaced this test with the following.  This should eliminate the dependence on faker data and make the test more robust to future changes.  Would you add a final commit with this or a similar update?

```js
  test('visiting a crate page from the front page', async function(assert) {
    this.server.create('crate', { id: 'nanomsg', newest_version: '0.6.1' });
    this.server.create('version', { crate: 'nanomsg', num: '0.6.1' });

    await visit('/');
    await click('[data-test-just-updated] [data-test-crate-link="0"]');

    assert.equal(currentURL(), '/crates/nanomsg/0.6.1');
    assert.equal(title(), 'nanomsg - crates.io: Rust Package Registry');

    assert.dom('[data-test-heading] [data-test-crate-name]').hasText('nanomsg');
    assert.dom('[data-test-heading] [data-test-crate-version]').hasText('0.6.1');
  });
```